src/Cache/FileCacheAdapter.php

```diff
@@ -0,0 +1,325 @@
+<?php
```

Comment:
I would much rather use someone else's cache implementation that write our own. We can require one in dev and suggest a list for production use, or we can require one so that our lib can be used without implementer's needing to add a cache dependency to their own apps.

Reply:
@jeremykendall agreed but if you suggest one in prod your update-psl script won't work out of the box because it would need a Cache implementation unless you specify one for production.